entity-framework/toc.md

```diff
@@ -29,7 +29,7 @@
 #### [ASP.NET Core](core/get-started/aspnetcore/index.md)
 ##### [ASP.NET Core - New Database](core/get-started/aspnetcore/new-db.md)
 ##### [ASP.NET Core - Existing Database](core/get-started/aspnetcore/existing-db.md)
-##### [EF Core Tutorial on ASP.NET Core site](https://docs.asp.net/en/latest/data/ef-mvc/intro.html)
+##### [EF Core Tutorial on ASP.NET Core site](/aspnet/core/data/ef-rp/intro)
```

Comment:
Should this mention it is razor pages or is that by far the version preferred by everyone?